examples_biomechanics/RebootMotionBiomechanicsSimulation_BatchInverseKineticsProcessing.ipynb

```diff
@@ -0,0 +1,1737 @@
+              "      border-top-color: var(--fill-color);\n",
+              "    }\n",
+              "    60% {\n",
+              "      border-color: transparent;\n",
+              "      border-right-color: var(--fill-color);\n",
+              "    }\n",
+              "    80% {\n",
+              "      border-color: transparent;\n",
+              "      border-right-color: var(--fill-color);\n",
+              "      border-bottom-color: var(--fill-color);\n",
+              "    }\n",
+              "    90% {\n",
+              "      border-color: transparent;\n",
+              "      border-bottom-color: var(--fill-color);\n",
+              "    }\n",
+              "  }\n",
+              "</style>\n",
+              "\n",
+              "  <script>\n",
+              "    async function quickchart(key) {\n",
+              "      const quickchartButtonEl =\n",
+              "        document.querySelector('#' + key + ' button');\n",
+              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
+              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
+              "      try {\n",
+              "        const charts = await google.colab.kernel.invokeFunction(\n",
+              "            'suggestCharts', [key], {});\n",
+              "      } catch (error) {\n",
+              "        console.error('Error during call to suggestCharts:', error);\n",
+              "      }\n",
+              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
+              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
+              "    }\n",
+              "    (() => {\n",
+              "      let quickchartButtonEl =\n",
+              "        document.querySelector('#df-7431651a-ed3a-437c-b927-73db4e5cf9a8 button');\n",
+              "      quickchartButtonEl.style.display =\n",
+              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
+              "    })();\n",
+              "  </script>\n",
+              "</div>\n",
+              "    </div>\n",
+              "  </div>\n"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 9
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "if not os.path.exists('player_data'):\n",
+        "    os.mkdir('player_data')"
+      ],
+      "metadata": {
+        "id": "gB4m4dj8K75t"
+      },
+      "execution_count": 10,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "roster_result_values = {}\n",
+        "for i in roster.index:\n",
+        "    player_meta_data = PlayerMetadata(\n",
+        "        org_player_ids=[roster.loc[i,'MLBAMID'].astype(str)],\n",
+        "        session_dates=None,\n",
+        "        session_nums=None,\n",
+        "        session_date_start=datetime.date(2023, 4, 1),\n",
```

Comment:
Could probably use the same widget that's in the other notebooks to select a date range, or make it a user input

Reply:
Added the widget. However, only the date range and year outputs are used. If desired I could make a new widget with just those inputs?